stream-chat-android-client/src/main/java/io/getstream/chat/android/client/channel/ChannelClient.kt

```diff
@@ -1,3 +1,5 @@
+@file:Suppress("DEPRECATION_ERROR")
```

Comment:
These are needed in a handful of places because we have to reference the types that we are marking as error level deprecations in this PR. File level annotation targest allow us to import these types despite using them being marked as an error.